app/stores.py

```diff
@@ -0,0 +1,25 @@
+from app.data_jason import load_json
+import dataclasses
+from typing import List
+
+
+@dataclasses.dataclass
+class Stores:
+    name: str
+    location: List[int]
+    products: dict
+
+    @classmethod
+    def get_stor(cls) -> list["Stores"]:
```

Comment:
```suggestion
    def load_stores(cls) -> list["Stores"]:
```

Reply:
`load_stores` is a better name
because you are load not one store but a list of them